src/Components/SmartComponents/Reports/Executive/__snapshots__/FirstPage.test.js.snap

```diff
@@ -152,294 +161,9 @@ servers.
-                        style={
-                          {
-                            "flex": 6,
-                          }
-                        }
-                      >
-                        <TEXT
-                          style={
-                            {
-                              "color": "#6a6e73",
-                              "fontSize": 9,
-                            }
-                          }
-                        >
-                           
-                        </TEXT>
-                        <VIEW
-                          style={
-                            [
-                              {
-                                "display": "flex",
-                                "flexDirection": "row",
-                                "justifyContent": "space-between",
-                              },
-                              {
-                                "justifyContent": "flex-start",
-                              },
-                            ]
-                          }
-                        >
-                          <CounterItem
-                            counter={11}
-                            label="Identified security rules"
-                          >
-                            <VIEW>
-                              <TEXT
-                                style={
-                                  {
-                                    "color": "#C9190B",
-                                    "fontSize": 20,
-                                  }
-                                }
-                              >
-                                11
-                              </TEXT>
-                              <TEXT
-                                style={
-                                  {
-                                    "fontSize": 9,
-                                  }
-                                }
-                              >
-                                Identified security rules
-                              </TEXT>
-                            </VIEW>
-                          </CounterItem>
-                        </VIEW>
-                      </VIEW>
-                    </PanelItem>
-                    <VIEW
-                      style={
-                        {
-                          "flex": 4,
-                        }
-                      }
-                    >
-                      <VIEW
-                        style={
-                          [
-                            {
-                              "flexDirection": "row",
-                              "justifyContent": "space-evenly",
-                            },
-                          ]
-                        }
-                      />
-                      <TEXT />
-                    </VIEW>
-                  </VIEW>
-                </VIEW>
-              </Panel>
-            </VIEW>
-          </VIEW>
+          <div />
```

Comment:
This is suspicious. Why is this chunk of the snapshot turning into just `div`?

Reply:
That is a result of the `frontend-components-pdf-generator`.

https://github.com/RedHatInsights/vulnerability-ui/blob/master/src/Components/SmartComponents/Reports/Executive/FirstPage.js#L47-L66

I'm not sure anything in my PR currently affect this.

Reply:
This mock is changing the test: https://github.com/RedHatInsights/vulnerability-ui/pull/1905/files#diff-b0b967f68ace8819a62c67421aebb1c7004922681bcbfd5aea1e03e4f64ed1e3R103-R113

The pdf-generator should have not been affected by any router changes. What was the reason for adding these mocks?